src/realm/parser/generated/query_flex.cpp

```diff
@@ -330,7 +331,7 @@ struct yy_buffer_state
 	/* Number of characters read into yy_ch_buf, not including EOB
 	 * characters.
 	 */
-	int yy_n_chars;
```

Comment:
The changes in these generated files are innocuous but unrelated to the rest of this PR. Probably we have slightly different tooling versions. I am using bison (GNU Bison) 3.7.4, and flex 2.6.4 for reference.

Reply:
I've tried to avoid committing the generated changes but it's pretty error-prone since the build system makes it impossible to not regenerate the files.

Reply:
Sure, no problem. I also noticed that there are new warnings reported for this file by Jenkins in the UWP build because of the changes so it might be worth while updating to the same version of the tools so that we have consistently generated code.